azure-pipelines.yml

```diff
@@ -177,3 +177,53 @@ stages:
     stageDependencies:
     - validate
     - qa
+- template: stage-with-burgr-notifications.yml@commonTemplates
+  parameters:
+    burgrName: 'dogfood'
+    burgrType: 'dogfood'
+    stageName: 'update_dogfood_repo'
+    stageDisplayName: Update internal dogfooding plugin repo
+    stageCondition: eq(variables['Build.SourceBranchName'], 'master')
```

Comment:
:+1: and we'll eventually use a dedicated branch if the frequency of dogfooding-related updates is too high :smile: